api/src/main/resources/emails/egress_remediation/content.html

```diff
@@ -23,8 +23,11 @@
 <body>
 <div>Hello ${FIRST_NAME},</div>
 <div>The ${ALL_OF_US} Researcher Workbench security team received an alert related to activity on your account <b>${USERNAME}</b>. </div>
+</div>
+${EGRESS_SOURCE}
+<div>
 <div>This alert was triggered because our monitoring system detected potential data egress from
-  your workspace when using the <span class="variable"><b>${ENVIRONMENT_TYPE}</b> application.</span>
+  your workspace<span class="variable">${EGRESS_SOURCE}</span>.
```

Comment:
why EGRESS_SOURCE is in line 27 and again in line 30?

Reply:
right, sorry, line 27 should be deleted